### PR TITLE
Fixes #685 (String resources needed for HistoryScanActivity)

### LINKED ISSUE #685: String resources needed for HistoryScanActivity
In the method (line 114) to export the history of the user, there are three headers that are not translated, "Barcode", "Name" and "Brands" along with the filename itself.

### PR DESCRIPTION
**Pull request by ManognaMutyala**
## Description

Getting values from string array as per the issue.

## Related issues and discussion
#685
 
 
 
 ## Checklist
 
 Please make sure these boxes are checked before submitting your pull request - thanks!
 
 
 
 - [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.
 
 - [ ] If you have multiple commits please combine them into one commit by squashing them.
 
 - [x] Read and understood the contribution guidelines .

### CODE REVIEW

**Comment by policorn** [comment]
Hi @ManognaMutyala, I've seen the missing strings are in #819. Please close that one (#819) and add the missing commit (c928dd47c873d699be298e320a1d0fcb0504e5dd) here.

**Comment by ManognaMutyala** [comment]
@policorn As you suggested I added the commits to the same branch!!

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/HistoryScanActivity.java [hunk 1]
```diff
@@ -131,7 +131,7 @@ public void exportCSV() {
             } else {
                 writer = new CSVWriter(new FileWriter(filePath));
             }
-            String[] headers = {"Barcode", "Name", "Brands"};
+           String[]  headers = res.getStringArray(R.array.headers);
```
Comment (policorn) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/HistoryScanActivity.java]: The build fails here: https://travis-ci.org/openfoodfacts/openfoodfacts-androidapp/builds/343922899#L2985

You should replace `res` by `getResources()` to make it work

**Comment by teolemon** [comment]
@ManognaMutyala a tiny fix so that we can merge that one :-)

**Comment by teolemon** [comment]
@policorn @ManognaMutyala merging

**Pull request merged**